### PR TITLE
Disable align to 0 of HexdumpWidget, highlight current seek (FIX #64)

### LINKED ISSUE #64: [TRACKER] Some HexdumpWidget UI features request
- [ ] Change format words, dwords, qwords, ..(pxh, pxw, pxq) on right click (see px?)
- [ ] Scrolling by line
- [ ] hex.compact
- [ ] hex.comments (pxc command)
- [ ] hex.pairs (it's set to false by default in cutter, but we should make it configurable for the user)
- [ ] Use same disassembly architecture in parsing panel as the one used by the opened file by default
- [ ] Hashes configurable in Hexdump -> information -> bytes info
- [ ] Endianess
- [ ] Colors hexdump
- [ ] Implement write operations (`wo?`) (see #715) 

---

- [x] Add ability to always align (like now aligned on 0) or never align (stand at current seek)
- [x] Show where current seek it
- [x] show flags (pxa command) (not compatible-yet with hex.compact radare/radare2#8957)
- [x] Hexdump is too wide #149
- [x] Place separators between areas to visually split them
- [x] Synchronize selection for hex-data and ASCII areas
- [x] Change number of columns on right click hex.cols
- [x] Add a button to close the side bar of the hexdump (see #636)
- [x] Add "right Click > Select Block" to the Hex Widget (see #761)

### PR DESCRIPTION
**Pull request by ITAYC0HEN**
Fixed and added features described in issue #64 .
Disabled the alginment to `0x???????0` and made the current seek the first column.
Added color to highlight the current seek address.

### CODE REVIEW

File: src/widgets/HexdumpWidget.cpp [hunk 1]
```diff
@@ -321,6 +318,12 @@ void HexdumpWidget::refresh(RVA addr)
     ui->hexHexText->setTextCursor(cursor);
     ui->hexHexText->ensureCursorVisible();
 
+    // Set the backgorund color of the current seek
+    QTextCursor offset_cursor(ui->hexOffsetText->document()->findBlockByLineNumber(curAddrLineOffset));
+    QTextBlockFormat format_tmp = offset_cursor.blockFormat();
+    format_tmp.setBackground(QColor(64, 129, 160));
+    offset_cursor.setBlockFormat(format_tmp);
+
```
Comment (xarkes) [inline src/widgets/HexdumpWidget.cpp]: Use camelCase please :)

**Comment by xarkes** [comment]
Thanks.

**Comment by radare** [comment]
I think the align to 0 should be an option.

**Comment by xarkes** [comment]
Agreed.

**Pull request merged**